### ISSUE TITLE
Required node version?

### ISSUE THREAD

**Issue by JuanCaicedo** [yoshuawuyts/markdown-to-medium#5]
I get the following error if I run this with node 4.4.5

```
/Users/JuanCaicedo/.nvm/versions/node/v4.4.5/lib/node_modules/markdown-to-medium/index.js:42
  let content = `
  ^^^

SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/Users/JuanCaicedo/.nvm/versions/node/v4.4.5/lib/node_modules/markdown-to-medium/bin/cli.js:12:14)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
```

However, I have no problems if I try this with node v6.9.1. 

Maybe the fix could be to specify node 6+ in the [engines](https://docs.npmjs.com/files/package.json#engines) field of `package.json`?

**Comment by RichardLitt** [comment]
Should be fixed by #4, too, I think.

**Comment by RichardLitt** [comment]
I can't replicate with v4.2.2, since #4 was merged. Closing this. Reopen if you have it again!
